miasm2/os_dep/linux_stdlib.py

```diff
@@ -36,13 +36,13 @@ def xxx_puts(jitter):
 
     writes the string s and a trailing newline to stdout.
     '''
-    ret_addr, (s,) = jitter.func_args_stdcall(1)
-    while True:
-        c = jitter.vm.get_mem(s, 1)
-        s += 1
-        if c == '\x00':
-            break
-        stdout.write(c)
+    ret_addr, args = jitter.func_args_stdcall(['s'])
```

Comment:
Are you sure that this is the better pattern for an equivalent of a `do... while` statement ?
It seems to introduce duplicated lines...

What about a pattern like the following ?
```Python
condition = True
while condition:
    do_stuff()
    condition = eval_cond()
```